corehq/apps/linked_domain/templates/linked_domain/partials/push_content.html

```diff
@@ -78,4 +78,12 @@ <h3>{% trans "Push Content" %}</h3>
       </div>
     </form>
   </div>
+  <div data-bind="if: !canPush()">
+    <p>
+      {% blocktrans %}
+        You have no linked project spaces that allow you to push data.<br/>
+        <a href="https://confluence.dimagi.com/display/commcarepublic/Linked+Project+Spaces" target="_blank">Learn more</a> about pushing content with Linked Project Spaces.
+      {% endblocktrans %}
+    </p>
+  </div>
```

Comment:
Is this a scenario that gets hit when dealing with an upstream domain that has only remote downstream domains?

Reply:
Yes. I think in most cases with remote downstreams, there aren't any local downstreams too.